recipe/build.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Run GLIBC check
+check-glibc bin/* lib/*.so.* lib64/*.so.*
```

Comment:
Is this ok for a check? Or do we need to do anything additional here?

Reply:
LGTM